compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/interop/UIKitView.uikit.kt

```diff
@@ -242,14 +252,16 @@ fun <T : UIView> UIKitView(
     onResize: (view: T, rect: CValue<CGRect>) -> Unit = DefaultViewResize,
     interactive: Boolean = true,
     accessibilityEnabled: Boolean = true,
+    areTouchesDelayed: Boolean = true
```

Comment:
I don't think that's another bool as 9th arg is a good idea here. I'd think about
- separate properties class
- specifying delay as ms or some mode as enum

Reply:
@Experimental composition local, perhaps?

Reply:
Honestly, I'd deprecate this entire thing before release and revisit the API. A lot of changes done after it was initially introduced break some expectations about the api.
`onResize` for example contains specifics for an unclipped frame of interop view inside a wrapper, for example.

Reply:
`update` and `background` don't also fit in there. Background could be assigned in a factory using native API (conversions are present), `update` is just something which I assume none should touch in a first place.

This function should in fact return a closure for user to allow changes on-demand but in a way synchronised with Compose rendering in case it's needed.

The changes themselves could be driven by any sort of `LaunchedEffect`s then.

Reply:
This point is not resolved. Adding such a parameter to non-experimental public API seems as a blocker for me

Reply:
Removed API change